src/bridge/transactions/peg_out.rs

```diff
@@ -14,6 +16,15 @@ pub struct PegOutTransaction {
     tx: Transaction,
     #[serde(with = "consensus::serde::With::<consensus::serde::Hex>")]
     prev_outs: Vec<TxOut>,
+    prev_scripts: Vec<Script>,
+}
+
+impl TransactionBase for PegOutTransaction {
+    fn tx(&mut self) -> &mut Transaction { &mut self.tx }
+
+    fn prev_outs(&self) -> &Vec<TxOut> { &self.prev_outs }
+
+    fn prev_scripts(&self) -> Vec<ScriptBuf> { self.prev_scripts.clone() }
```

Comment:
@franklynot , I merged this for now. I think when we do a larger refactor we can remove this if it's not needed. For now, since both PegOut and Kickoff follow the prev_out + prev_scripts convention I sort of see TransactionBase as just that - a place where we might define more of a public interface.